test/y2storage/filesystems/btrfs_test.rb

```diff
@@ -639,4 +639,22 @@
       end
     end
   end
+
+  describe "#default_subvolume_name" do
+    context "when a default subvolume name is set to '@'" do
+      let(:dev_name) { "/dev/sda2" }
+
+      it "returns the default subvolume name '@'" do
+        expect(subject.default_subvolume_name).to eq("@")
+      end
+    end
+
+    context "when default subvolume name is not set" do
+      let(:dev_name) { "/dev/sdd1" }
+
+      it "returns an empty string" do
+        expect(subject.default_subvolume_name).to eq("")
+      end
+    end
+  end
```

Comment:
I would add one more tests where the `default_btrfs_subvolume.path` and `default_subvolume_name` are different, to clarify that they are not related. For example, with snapshots, the `default_btrfs_subvolume.path` would be `@/.snapshots`, isn't it?

Reply:
It would be `@/.snapshots/1/snapshot` (or whatever snapshot it is the current one). And the `default_subvolume_name` could be `@`.

For instance, the second test checks that `default_subvolume_name` is not set and, although subvolumes are named like `@/blah`, the `default_subvolume_name` is still `""`.